src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 13px;
```

Comment:
As i remeber the stars have a width and height of 16 px in figma mockup, plese change